diff -up lxqt-qtplugin-2.3.0/src/lxqtplatformtheme.cpp.1~ lxqt-qtplugin-2.3.0/src/lxqtplatformtheme.cpp
--- lxqt-qtplugin-2.3.0/src/lxqtplatformtheme.cpp.1~	2025-11-05 12:32:44.000000000 +0100
+++ lxqt-qtplugin-2.3.0/src/lxqtplatformtheme.cpp	2025-11-05 18:44:25.247864980 +0100
@@ -100,7 +100,7 @@ void LXQtPlatformTheme::loadSettings() {
     settingsFile_ = settings.fileName();
 
     // icon theme
-    iconTheme_ = settings.value(QLatin1String("icon_theme"), QLatin1String("oxygen")).toString();
+    iconTheme_ = settings.value(QLatin1String("icon_theme"), QLatin1String("breeze")).toString();
     iconFollowColorScheme_ = settings.value(QLatin1String("icon_follow_color_scheme"), iconFollowColorScheme_).toBool();
 
     // read other widget related settings form LxQt settings.
@@ -250,10 +250,10 @@ void LXQtPlatformTheme::loadSettings() {
     settings.beginGroup(QLatin1String("Qt"));
 
     // widget style
-    style_ = settings.value(QLatin1String("style"), QLatin1String("fusion")).toString();
+    style_ = settings.value(QLatin1String("style"), QLatin1String("Breeze")).toString();
 
     // SystemFont
-    fontStr_ = settings.value(QLatin1String("font")).toString();
+    fontStr_ = settings.value(QLatin1String("font"), QLatin1String("Noto Sans,11,-1,5,400,0,0,0,0,0,0,0,0,0,0,1")).toString();
 
     if(!fontStr_.isEmpty()) {
         if(font_.fromString(fontStr_)) {
@@ -264,7 +264,7 @@ void LXQtPlatformTheme::loadSettings() {
     }
 
     // FixedFont
-    fixedFontStr_ = settings.value(QLatin1String("fixedFont")).toString();
+    fixedFontStr_ = settings.value(QLatin1String("fixedFont"), QLatin1String("Liberation Mono,11,-1,5,400,0,0,0,0,0,0,0,0,0,0,1")).toString();
     if(!fixedFontStr_.isEmpty()) {
         fixedFont_.fromString(fixedFontStr_);
     }
@@ -280,7 +280,7 @@ void LXQtPlatformTheme::loadSettings() {
     // mouse cursor
     QSettings sessionSettings(QSettings::UserScope, QLatin1String("lxqt"), QLatin1String("session"));
     sessionSettings.beginGroup(QStringLiteral("Mouse"));
-    mouseCursorTheme_ = sessionSettings.value(QLatin1String("cursor_theme"));
+    mouseCursorTheme_ = sessionSettings.value(QLatin1String("cursor_theme"), QLatin1String("breeze_cursors"));
     int curSize = sessionSettings.value(QLatin1String("cursor_size"), 24).toInt();
     mouseCursorSize_ = QSize(curSize, curSize);
     sessionSettings.endGroup();
